README.md

```diff
@@ -37,7 +37,7 @@ Run the ``deploy.sh`` script within a tmux or screen session:
 tmux
 cd /opt/rpc-openstack
 export DEPLOY_AIO=true
-export RPC_PRODUCT_RELEASE="queens"  # This is optional, if unset the current stable product will be used
+export RPC_PRODUCT_RELEASE="master"  # This is optional, if unset the current stable product will be used
```

Comment:
All these 'master' references need to match the entry in ``playbooks/vars/rpc-release.yml``, so they should be ``rocky`` rather than ``master``.

Reply:
Given the changes to make RPC_PRODUCT_RELEASE to 'master', this comment is no longer valid.